packages/frontend/editor-ui/src/components/Node/NodeCreator/utils.ts

```diff
@@ -221,3 +227,76 @@ export const formatTriggerActionName = (actionPropertyName: string) => {
 	}
 	return changeCase.noCase(name);
 };
+
+export const removePreviewToken = (key: string) =>
+	key.replace(COMMUNITY_NODE_TYPE_PREVIEW_TOKEN, '');
+
+export const isNodePreviewKey = (key = '') => key.includes(COMMUNITY_NODE_TYPE_PREVIEW_TOKEN);
+
+export function extendItemsWithUUID(items: INodeCreateElement[]) {
+	return items.map((item) => ({
+		...item,
+		uuid: `${item.key}-${uuidv4()}`,
+	}));
+}
+
+export const getMoreFromCommunity = (
```

Comment:
Naming: this is not really anything related to community nodes since you pass in the nodes to search